api/models/manifest.go

```diff
@@ -532,3 +532,69 @@ func (me ManifestEntry) RegistryImage(app *App, buildId string) string {
 
 	return fmt.Sprintf("%s/%s-%s:%s", os.Getenv("REGISTRY_HOST"), app.Name, me.Name, buildId)
 }
+
+type MapOrEqualSlice []string
+
+// UnmarshalYAML implements the Unmarshaller interface.
+func (s *MapOrEqualSlice) UnmarshalYAML(unmarshal func(interface{}) error) error {
```

Comment:
this is the only interesting bit, changing the Unmarshaller interface from what libcompose does to what gopkg.in does.